ansible-operator.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: ansible-operator
+  version: 1.36.1
+  epoch: 0
+  description: Ansible operator to run in kubernetes environments.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - py3-ansible-core
+      - py3-ansible-runner-http
+      - py3-kubernetes
+      - py3-pip
+      - py3-pipenv
+      - python3
```

Comment:
It only supports latest Python?

Reply:
Ah wait, we can still drop in whatever as other versions are provided... Don't mind me